src/components/Screen.js

```diff
@@ -1,10 +1,10 @@
-import KeyPad from './Calculator';
+// import Calculator from './Calculator';
 
-const Display = () => (
-  <div className="display">
-    <p className="screen">0</p>
-    <KeyPad />
-  </div>
+// const Display = () => (
+//   <div className="display">
+//     <p className="screen">0</p>
+//     <Calculator />
+//   </div>
```

Comment:
**[OPTIONAL BUT NECESSARY FOR IMPROVEMENT]** Please consider removing this commented code as you are no longer using it and it has a bad impact on your professional repo and skills.